tools/sgen/input/FluentUIStyle.yml

```diff
@@ -265,11 +265,22 @@ AP_MSFListCellTokens:
   labelFont: $Typography.body
   leadingViewColor: $Colors.Foreground.neutral4
   leadingViewSize: [ small: $Icon.size.xSmall, medium: $Icon.size.medium, large: $Icon.size.xxlarge ]
+  sublabelAccessorySize: $Icon.size.xxSmall
   sublabelColor: $Colors.Foreground.neutral3
   sublabelFont: $Typography.subheadline
   trailingItemForegroundColor: $Colors.Foreground.neutral3
   trailingItemSize: $Icon.size.medium
 
+MSFPersonaTokens extends MSFListCellTokens:
+  cellHeight: [ twoLines: 84pt]
```

Comment:
Can't we just set the layoutType to .threeLines to set the default height of 84pt instead of doing this override?